applications/machine_learning/README.rst

```diff
@@ -207,6 +207,9 @@ Buttons
 The application supports a button that is used to switch between data forwarding and running the machine learning model.
 You can trigger the change by pressing and holding the button for longer than 5 seconds.
 
+.. note::
+   If a given configuration supports Bluetooth, then pressing a button shortly after application boot triggers removing Bluetooth bond.
```

Comment:
The note is not clear that which button and what exactly happens when the button is pressed.

Reply:
Refered to the button as `the button`. There is only one button used to control app state. Is it ok now ?